### PR TITLE
Badge Component Added

### PR DESCRIPTION
**Pull request by Mahi24-prog**
### Badge Component Added
Types :

- Badge on Icon

- Badge on Avatar

### Review Link 
https://badgeui.netlify.app/

### CODE REVIEW

File: component/badge.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
+#badge-component{
+    display: flex; 
+    flex-direction: column;
+ }
+ 
+ 
+ 
+ #badge-component #badge-onicon{
+     
+ }
```
Comment (fahd23) [inline component/badge.css:10]: for what this id is for?

File: component/badge.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
+#badge-component{
+    display: flex; 
+    flex-direction: column;
+ }
+ 
+
```
Comment (vaibhav18matere) [inline component/badge.css:6]: remove extra spaces

File: component/badge.css [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    display: flex; 
+    flex-direction: column;
+ }
+ 
+ 
+ 
+ #badge-component #badge-onicon{
+     
+ }
+
```
Comment (vaibhav18matere) [inline component/badge.css:11]: remove un-used classes, ids

**Review by fahd23 — CHANGES_REQUESTED** [review]

File: component/badge.css [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+ .badge{
+     display: inline-block;
+     position: relative;
+     margin: 1rem 4rem;
+ }
+ 
+ .badge .number{
+     display: flex;
+     justify-content: center;
+     align-items: center;
```
Comment (vaibhav18matere) [inline component/badge.css:21]: as this is repeating, you can make separate css class for it and add in html class
for ex. d-flex, d-center etc.

File: component/badge.css [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+     display: inline-block;
+     position: relative;
+     margin: 1rem 4rem;
+ }
+ 
+ .badge-avatar .status{
+     display: flex;
+     justify-content: center;
+     align-items: center;
+     color: white;
```
Comment (vaibhav18matere) [inline component/badge.css:63]: display-flex is repeating, make separate class

File: component/badge.css [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+     background-color: #576574;
+ }
+ 
+ .badge-success{
+     background-color: #5cb85c;
+ }
+ 
+ .badge-danger{
+     background-color: #d9534f;
+ }
```
Comment (vaibhav18matere) [inline component/badge.css:45]: make separate css variables for success, danger, primary and secondary colors in main.css/home.css/reset.css file

File: component/badge.css [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+     height: 1.8rem;
+     width: 1.8rem;
+     border-radius: 50%;
+     position: absolute;
+     bottom: 1.5rem;
+     right: 2rem;
+ }
+ 
+ .badge-avatar .status-online{
+     background-color: green;
```
Comment (vaibhav18matere) [inline component/badge.css:103]: make seperate success css class and add -
background color:var(--success);

File: component/badge.css (hunk 2 above)
Comment (Mahi24-prog) [reply component/badge.css:6]: ok

File: component/badge.css (hunk 4 above)
Comment (Mahi24-prog) [reply component/badge.css:21]: ok

File: component/badge.css (hunk 6 above)
Comment (Mahi24-prog) [reply component/badge.css:45]: Color variable added

File: component/badge.css (hunk 7 above)
Comment (Mahi24-prog) [reply component/badge.css:103]: color variables added

**Pull request merged**